src/AnalyzeRegistry.jl

```diff
@@ -130,6 +122,59 @@ function Base.show(io::IO, p::Package)
     print(io, strip(body))
 end
 
+function ci_services(p)
+    return (p.github_actions => "GitHub Actions",
+            p.travis => "Travis",
+            p.appveyor => "AppVeyor",
+            p.cirrus => "Cirrus",
+            p.circle => "Circle",
+            p.drone => "Drone CI",
+            p.buildkite => "Buildkite",
+            p.azure_pipelines => "Azure Pipelines",
+            p.gitlab_pipeline => "GitLab Pipeline")
+end
+
+emojify(b::Bool) = b ? "☑" : "◻"
+
+function print_check(io, check, name, parens)
+    print(io, emojify(check), " ", name)
+    if check && !isempty(parens)
+        println(io, " (", parens, ")")
+    else
+        println(io)
+    end
+end
+
+healthcheck(p::Package) = healthcheck(stdout, p)
+
+function healthcheck(io::IO, p::Package)
```

Comment:
If the idea is to have something similar also for RegistryCI, perhaps we want to have multiple methods for different MIMEs (plain text vs markdown)?